src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  color: #3d4e61;
+}
+
+body {
+  margin-top: 0;
+}
+
+.searchingForm {
+  position: relative;
+  z-index: 1;
+}
+
+.searchingBar {
+  width: 100%;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  position: absolute;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
```

Comment:
always set font styles to the `html` tag and inherit where needed

Reply:
it doesn't work in html tag